templates/azure/terraform/expand_property_method.erb

```diff
@@ -7,7 +7,36 @@
                            prefix: sdk_marshal.resource,
                            property: property)) -%>
 <%  else -%>
-<%    if property.is_a?(Api::Type::Map) -%>
+<%    if property.is_a?(Api::Type::Array) && property.item_type == 'Api::Type::Integer'
+      bit_number = 32 if sdk_marshal.sdktype.type_definition.is_a?(Api::Azure::SDKTypeDefinition::Integer32ArrayObject)
+      bit_number = 64 if sdk_marshal.sdktype.type_definition.is_a?(Api::Azure::SDKTypeDefinition::Integer64ArrayObject)
+-%>
+func expandIntegerArray(input []interface{}) *[]int<%= bit_number -%> {
```

Comment:
As we talked, we can make this a common function.

Reply:
Done.